src/routes/cart.router.js

```diff
@@ -0,0 +1,14 @@
+const express = require("express");
+const cartController = require("../controllers/cart.controller");
+const { verifyUser } = require("../utils/loginRequired");
+const cartRouter = express.Router();
+
+cartRouter.post("/insertion", verifyUser, cartController.createCart);
+
+cartRouter.patch("/edition", verifyUser, cartController.editCart);
+
+cartRouter.delete("/deletion", verifyUser, cartController.deleteCart);
+
+cartRouter.get("/retrieval", verifyUser, cartController.getCart);
```

Comment:
다시 한 번 강조하지만 RESTful API에서 엔드포인트에는 동사를 포함하지 않습니다.
동명사, 동사가 명사로 대체되는 것도 마찬가지입니다.

```suggestion
cartRouter.post("/", verifyUser, cartController.createCart);
cartRouter.patch("/", verifyUser, cartController.editCart);
cartRouter.delete("/", verifyUser, cartController.deleteCart);
cartRouter.get("/", verifyUser, cartController.getCart);
```